src/api/protectedApiClient.ts

```diff
@@ -52,11 +57,22 @@ const makePFRequest = (): Promise<void> => {
     .catch((err) => err);
 };
 
+const getEventAnnouncements = (
+  eventId: number,
+): Promise<EventAnnouncement[]> => {
+  return AppAxiosInstance.get(
+    `${ProtectedApiClientRoutes.ANNOUNCEMENTS}/${eventId}`,
+  )
+    .then((res) => res.data?.announcements)
```

Comment:
Why use the ? here? If announcements is undefined then we want to error right?

Reply:
agree